code/game/machinery/vendors/generic_vendors.dm

```diff
@@ -929,6 +929,7 @@
 					/obj/item/food/spacetwinkie = 6,
 					/obj/item/food/cheesiehonkers = 6,
 					/obj/item/food/tastybread = 6,
+					/obj/item/food/candy/chocolate_bar = 6,
```

Comment:
You intend to add this non junk food to this vendor? Far as i know the only non junk food available is the syndicake, as a contraband

Reply:
![image](https://github.com/user-attachments/assets/c3483086-f2a7-412b-92f8-6b2d5a864e1b)

Almost all junk food use this, which limits how much of certain food one can eat in a row from what i can remember.

It depends on what you want this chocolate bar to be, a junk food that isnt good for your body or a premium one that you would eat in a restaurant.